gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -177,6 +177,40 @@ def add_arguments(self, parser):
               'See the "Complext VCF Entries" of this doc for details:'
               'http://www.ensembl.org/info/docs/tools/vep/online/'
               'VEP_web_documentation.pdf'))
+    parser.add_argument(
+        '--run_annotation_pipeline',
+        type='bool', default=False, nargs='?', const=True,
+        help=('If true, runs annotation tools (currently only VEP) on input '
+              'VCFs before loading to BigQuery.'))
+    parser.add_argument(
+        '--vep_image_uri',
+        default="",
+        help=('The URI of the docker image for VEP.'))
+    parser.add_argument(
+        '--vep_cache_path',
+        default="",
+        help=('The path for VEP cache on Google Cloud Storage.'))
+    parser.add_argument(
```

Comment:
nit: please move this arg to the top to make all VEP-specific args grouped together.

Reply:
Done.